### ISSUE TITLE
Not sending notifications for new results if user is both clinic worker and CBA

### ISSUE THREAD

**Issue by tsinkala** [mwana/mwana#59]
The system does not send notifications for new results if a user is both a clinic worker and a CBA

**Comment by tobiasmcnulty** [comment]
that sounds bad.  is it possible to reproduce in a unit test (and subsequently fix)?

**Comment by merrick** [comment]
wow I agree.  this should be a very high priority fix...would it be better to make people only be able to registered at a clinic worker OR as a CBA?

**Comment by tsinkala** [comment]
I think they should just be registered as one of CBA and clinic worker

**Comment by merrick** [comment]
We debated this during the September 2nd Product Backlog Meeting:

Should a user in the system be allowed to be both a CBA and Clinic Worker?
plus: 
*the roles may blend in the real world,
*it is less work to fix the bug

minus: 
*sending messages to groups could get confusing if people are in multiple groups
*this could be an accidental state and thus a real usability issue

We decided to fix this bug directly, ie. make sure the that if someone is registered as both a CBA and a Clinic Worker, that they can receive messages.

We also agreed to make the larger usability issue a separate issue.
